docs/pages/management/guides/joining-nodes-azure.mdx

```diff
@@ -0,0 +1,137 @@
+your Teleport cluster needs a Managed Identity assigned to it. The identity 
+requires at least the `Microsoft.Compute/virtualMachines/read` permission.
+
+To create a managed identity with an appropriate role, first
+[create a custom role](https://learn.microsoft.com/en-us/azure/role-based-access-control/custom-roles)
+with the `Microsoft.Compute/virtualMachines/read` permission.
+Then [create a new user-assigned managed identity](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-manage-user-assigned-managed-identities?pivots=identity-mi-methods-azp#create-a-user-assigned-managed-identity)
+and [assign it your custom role](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-manage-user-assigned-managed-identities?pivots=identity-mi-methods-azp#manage-access-to-user-assigned-managed-identities).
+Note the client ID of the identity for Step 3. Finally,
+[assign the identity to the VM that will join your Teleport cluster](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/qs-configure-portal-windows-vm).
+
+<Admonition
+  type="warning"
+  title="IMPORTANT"
+>
+As part of authentication, Teleport will be able to do anything allowed by the
+identity. We strongly recommend limiting the managed identity's permissions
+to only what is necessary. 
+</Admonition>
+
+## Step 2/4. Create the Azure Node joining token
+Under the hood, Nodes will prove that they are running in your Azure subscription
+by sending a signed attested data document and access token to the Teleport Auth
+server. The Node's identity must match an allow rule configured in your Azure
+Node joining token.
+
+Create the following `token.yaml` with an `allow` rule specifying your Azure
+subscription and the resource group that your Node's identity must match.
+
+```yaml
+# token.yaml
+kind: token
+version: v2
+metadata:
+  # the token name is not a secret because instances must prove that they are
+  # running in your Azure subscription to use this token
+  name: azure-token
+  # set a long expiry time, the default for tokens is only 30 minutes
+  expires: "3000-01-01T00:00:00Z"
+spec:
+  # use the minimal set of roles required
+  roles: [Node]
+  azure:
+    allow:
+      # specify the Azure subscription which Nodes may join from
+      - subscription: 11111111-1111-1111-1111-111111111111
+      # multiple allow rules are supported
+      - subscription: 22222222-2222-2222-2222-222222222222
+      # resource_groups is optional and allows you to restrict the resource group of joining Nodes
+      - subscription: 33333333-3333-3333-3333-333333333333
+        resource_groups: ["group1", "group2"]
+```
+
+The token name `azure-token` is just an example and can be any value you want to
+use, as long as you use the same value for `join_params.token_name` in Step 3.
+
+Run `tctl create -f token.yaml` to create the token.
```

Comment:
````suggestion
Run the following command  to create the token:

```code
$ tctl create -f token.yaml
```
````

So a user can copy/paste this more easily